docs/cloud-common.md

```diff
@@ -0,0 +1,87 @@
+# Set up Ark with your cloud provider
+
+To run Ark with your cloud provider, you specify provider-specific settings for the Ark server. In version 0.7.0 and later, you can run Ark in any namespace, which requires additional customization. See [Run in custom namespace][3].
+
+The Ark repository includes a set of example YAML files that specify the settings for each cloud provider. See the following for provider-specific instructions:
+
+* [AWS][0]
+* [GCP][1]
+* [Azure][2]
+
+## Examples
+
+After you set up the Ark server, try these examples:
+
+### Basic example (No PVs)
+
+Start the sample nginx app:
+
+```bash
+kubectl apply -f examples/nginx-app/base.yaml
+```
+
+Now create a backup:
+
+```bash
+ark backup create nginx-backup --include-namespaces nginx-example
+```
+
+Simulate a disaster:
+
+```bash
+kubectl delete namespaces nginx-example
+```
+
+Now restore your lost resources:
+
+```bash
+ark restore create nginx-backup
+```
+
+### Snapshot example (With PVs)
+
+> NOTE: For Azure, your Kubernetes cluster needs to be version 1.7.2+ in order to support PV snapshotting of its managed disks.
+
+Start the sample nginx app:
+
+```bash
+kubectl apply -f examples/nginx-app/with-pv.yaml
+```
+
+Now create a backup with PV snapshotting:
+
+```bash
+ark backup create nginx-backup --include-namespaces nginx-example
+```
+
+Simulate a disaster:
+
+```bash
+kubectl delete namespaces nginx-example
+kubectl delete pv $nginx_pv_name
+```
+
+Because the default [reclaim policy][19] for dynamically-provisioned PVs is "Delete", the above commands should trigger your cloud provider to delete the disk backing the PV. The deletion process is asynchronous so this may take some time. **Before continuing to the next step, check your cloud provider (via dashboard or CLI) to confirm that the disk no longer exists.**
+
+Now restore your lost resources:
+
+```bash
+ark restore create nginx-backup
+```
+
+## Expired backup deletion
```

Comment:
This section and the following seem out of place on this page

Reply:
Agreed